pylint/extensions/check_docs.py

```diff
@@ -56,97 +46,77 @@ class ParamDocChecker(BaseChecker):
         'W9005': ('"%s" has constructor parameters documented in class and __init__',
                   'multiple-constructor-doc',
                   'Please remove parameter declarations in the class or constructor.'),
+        'W9006': ('Raising a "%s" is not documented',
+                  'missing-raise-doc',
+                  'Please document that this exception type is raised.'),
     }
 
     options = (('accept-no-param-doc',
                 {'default': True, 'type' : 'yn', 'metavar' : '<y or n>',
                  'help': 'Whether to accept totally missing parameter '
                          'documentation in a docstring of a function that has '
-                         'parameters'
+                         'parameters.'
+                }),
+               ('accept-no-raise-doc',
+                {'default': True, 'type' : 'yn', 'metavar' : '<y or n>',
+                 'help': 'Whether to accept totally missing raises'
+                         'documentation in a docstring of a function that'
+                         'raises an exception.'
                 }),
               )
 
     priority = -2
 
+    constructor_names = set(["__init__", "__new__"])
```

Comment:
By the way, we don't support Python < 2.7, so these could as well be ``{'_init_', '_new_'}``